### ISSUE TITLE
reverseRelation does not work?

### ISSUE THREAD

**Issue by USvER** [PaulUithol/Backbone-relational#493]
Week HasMany Days with reverseRelation
I want to get Week from any Day, but `day.get('week')` returns `null`

So the code is:

``` javascript
var Day = Backbone.RelationalModel.extend({
  initialize: function(){
    alert(this.get('week')); // Alerts null but should be 'first' week
  }
});
var Week = Backbone.RelationalModel.extend({
  relations: [{
    type: Backbone.HasMany,
    key: 'days',
    relatedModel: Day,
    reverseRelation: {
      key: 'week',
    }
  }]
});
first = new Week;
first.get('days').create({name: "Monday"});
```

If reverseRelation is not meant to work like that, how can i get this functionality?
PS: Here is the code http://jsfiddle.net/77rmbevw/1/

**Comment by RalphSleigh** [comment]
I have a similar issue:

1) Marionette composite view listening to an initially empty hasMany collection
2) Call fetch on the collection to load some related models from server
3) The models come, get added to the collection
4) add triggered on collection, causing the view to be rendered.
5) Inverse relation is then set on the new models

My problem is that my view depends on the inverse relation to pull a field from the parent model, so, #5 needs to happen before #4. 

Current workaround plan is when I call fetch, stash the current parent model in some global state then get it out in an event attached to my new models being added to the collection.

TLDR: Everything should be in place before views render anything, currently not.
